src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
move Random to the class field, like
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/dd266c65-ca0a-4d84-84e8-5870fe6db995)